tests/models/bloom/test_modeling_bloom.py

```diff
@@ -396,7 +397,7 @@ def test_simple_generation(self):
         # >=760m + allow_fp16_reduced_precision_reduction = False  + torch.bmm  ==> PASS
 
         path_350m = "bigscience/bloom-350m"
-        model = BloomForCausalLM.from_pretrained(path_350m, use_cache=True).cuda()
+        model = BloomForCausalLM.from_pretrained(path_350m, use_cache=True, revision="gs555750").cuda()
```

Comment:
What does the `revision` argument do? Are we sure we want to add that?

Reply:
yes, this argument loads the model from the initial model we were using when designing those tests: https://huggingface.co/bigscience/bloom-350m/tree/gs555750 
@Muennighoff recently wanted to push the newest weights of 350m so just adding the revision flag should prevent us breaking those tests